cev_eris.dme

```diff
@@ -993,6 +992,15 @@
 #include "code\game\objects\items\weapons\circuitboards\machinery\telecomms.dm"
 #include "code\game\objects\items\weapons\circuitboards\machinery\unary_atmos.dm"
 #include "code\game\objects\items\weapons\circuitboards\machinery\vending.dm"
+#include "code\game\objects\items\weapons\design disks\_disk.dm"

```

Comment:
There's a space here, should be `design_disks`, good luck making that change though since Windows File Explorer won't recognize a change like this, unless you use LINUX.